src/test/java/game/map/MapValidatorTest.java

```diff
@@ -0,0 +1,256 @@
+package game.map;
+
+import static game.map.MapLoader.loadMap;
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+
+import game.map.MapValidator;
+import game.pojo.Continent;
+import game.pojo.Country;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class MapValidatorTest {
+	
+	private String d_path = "src/test/resources/test_load_map.map";
+	private Map d_map;
+    Continent d_continent1 = new Continent(1, "Norddeutschland", 3);
+    Continent d_continent2 = new Continent(2, "Westdeutschland", 4);
```

Comment:
I think we can maybe avoid initializing them here as we are doing that in `setUp()` method.